microsoft-edge/index.yml

```diff
@@ -508,16 +509,16 @@ additionalContent:
 # Card
         - summary: Get advice and answers from community experts.
           title: Community Support on Microsoft Q&A
-          url: /answers/topics/ms-edge.html
+          url: /answers/topics/ms-edge.html  # https://learn.microsoft.com/answers/tags/781/microsoft-edge
 # -----------------------------------------------------------------------------
 # Card
-        - summary: If your business needs immediate technical support for Microsoft Edge, 1:1 support from Microsoft is available using a variety of plans to help you get your required level of support.
-          title: Microsoft Support for Business
+        - summary: If your business needs immediate technical support for Microsoft Edge, 1-on-1 support from Microsoft is available using a variety of plans to help you get your required level of support.
+          title: Microsoft Support for business
           url: https://support.microsoft.com/supportforbusiness/productselection?sapId=a77ee9b7-b6b6-aa08-d7b9-887ebe228207
 # -----------------------------------------------------------------------------
 # Card
-        - summary: Helpful support resources for users is available.
-          title: User Support
+        - summary: Helpful support resources for end-users.
```

Comment:
the `summary:` text can be improved, when you look at rendered hub page.  All devs and users need end user docs; to develop a feature, you can read end user docs at SMC about the feature.  
The end user POV at SMC is for end users and for Devs.
@captainbrosset

Reply:
result:
```
        - summary: Support for end users and Devs about how to use Microsoft Edge features.
          title: Microsoft Edge help & learning
          url: https://support.microsoft.com/microsoft-edge
```